code/game/objects/effects/step_triggers.dm

```diff
@@ -163,12 +163,12 @@
 		s.start()
 
 	if(entersmoke)
-		var/datum/effect_system/smoke_spread/s = new
-		s.set_up(4, 1, src, 0)
+		var/datum/effect_system/fluid_spread/smoke/s = new
+		s.set_up(DIAMOND_AREA(4), src)
```

Comment:
is there ever a case where you won't want to pass a diamond area into fluid spread setup?

Reply:
The chem reactions pass a multiple of the reaction volume directly.

Reply:
Resolved in [b3c6509](https://github.com/tgstation/tgstation/pull/65281/commits/b3c6509d40b00981ff5c2ce3bc27c4ce744630f0)